agent/agent.go

```diff
@@ -71,16 +71,17 @@ func (a *Agent) Routes() common.Routes {
 			func() interface{} {
 				return &NetIf{}
 			},
+			false,
 		},
 	}
 	return routes
 }
 
 // Run runs the agent service.
```

Comment:
Maybe to avoid repetition, we could say: "Run starts the agent service."
Also, a few comments within that function would be nice: A 'helper' is created, what is that for? A client is created, just to retrieve the config. Where is the config? Etc.